test/test_ops.py

```diff
@@ -236,22 +236,22 @@ def test_expand(self):
   def test_sd_big_conv(self):
     # internal shape (1, 1, 512, 62, 62, 512, 3, 3) overflows a int
     helper_test_op([(1,256,64,64), (512,256,3,3)],
-                    lambda x,w: torch.nn.functional.conv2d(x, w),
+                    torch.nn.functional.conv2,
```

Comment:
broken

Reply:
this functions the same, and satisfies the linter check for unnecessary lambda

Reply:
no it doesn't, please read carefully

Reply:
typo, fixed it and added a conditional to the the skip for ALL env so it can easily be run without having to call the function directly